doc/remailers.rst

```diff
@@ -0,0 +1,129 @@
+Autocrypt and Mailing Lists
+====
+
```

Comment:
NOTE: this document and the accompanying discussions on the https://github.com/autocrypt/autocrypt/pull/374  according pull request reflect the unresolved state of discussions in 2018 around the topic of remailers and Autocrypt integration.  Several people intend to revisit the interesting question of autocrypt/remailer integration when the times comes for it.  If you'd like to do a project towards this end please try engage through Autocrypt channels (IRC and ML at this point, as well as physical gatherings from time to time, also in 2019).